src/Modules/Admin/Commands.Admin/help/Add-PowerBIEncryptionKey.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.PowerBI.Commands.Admin.dll-Help.xml
+Module Name: MicrosoftPowerBIMgmt.Admin
+online version:
+schema: 2.0.0
+---
+
+# Add-PowerBIEncryptionKey
+
+## SYNOPSIS
+{{Fill in the Synopsis}}
+
+## SYNTAX
+
+```
+Add-PowerBIEncryptionKey -Name <String> -KeyVaultKeyUri <String> [-Default <Boolean>] [-Activate <Boolean>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+{{Fill in the Description}}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
+
+## PARAMETERS
+
+### -Activate
+{{Fill Activate Description}}
```

Comment:
Fill in, repeat for rest of file where there is {{XXXX}}

Reply:
Will get the exact strings from pm and update it for public preview. Temporarily filled in with the spec strings